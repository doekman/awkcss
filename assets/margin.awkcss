BEGIN {
	width(6);
	margin();
}
NR == 1 {
	margin(1);
}
NR == 2 {
	margin(1, 2);
}
NR == 3 {
	margin(1, 2, 3);
}
NR == 4 {
	margin(1, 2, 3, 4);
}
