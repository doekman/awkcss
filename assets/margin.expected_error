‼️ margin value '' is not recognized and will be ignored
‼️ margin_left with value '10': Ignored, because width is too small.
‼️ margin_right with value '9': Ignored, because width is too small.
