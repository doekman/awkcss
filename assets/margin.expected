 Line 1   
  Line 2  
   Line   
    3     
    Li    
    ne    
     4    
Line 5    
 Line 6   
